### ISSUE TITLE
Scirius dashboard and some elk dashboards are not showing after modify ES data path.

### ISSUE THREAD

**Issue by JINN00** [StamusNetworks/SELKS#126]
Hi,

I installed SELKS and applied some updates. up to this point, all the dashboards worked well.
Then, I wanted to divide the elasticsearch's data path because elasticsearch data takes up a lot of capacity.
So, I replaced elasticsearch's data path to other disk like below.
DATA_DIR=/var/lib/elasticsearch
DATA_DIR=/data/elasticsearch

Since then, data from elasticsearch has been storing well but some dashboards from scirius and ELK are not showing like below pictures
![image](https://user-images.githubusercontent.com/20368801/43943354-3d3005e6-9cb6-11e8-84b8-f3f544bdb988.png)
![image](https://user-images.githubusercontent.com/20368801/43943391-6416552a-9cb6-11e8-8a77-0b8b9dfa6ec2.png)

If I apply default elasticsearch's data path again, it's showing well.
Which settings should be applied to change the elasticsearch's data path and showing dashboard well?

**Comment by pevma** [comment]
Where did you replace that ?

**Comment by JINN00** [comment]
I modified file which /etc/default/elasticsearch

And, there was a problem that the date of the logstash data stored in elasticsearch is not updated after replace data path and restart elasticsearch. 
So I restarted logstash and the problem which can not see dashboard is solved.
The date of logstash data are also well represented by the current date.

But I can not sure if logstash was the cause of the problem.
If logstash disconnected with elasticsearch awhile, could happen like this?
